statesync/reactor.go

```diff
@@ -34,7 +34,7 @@ var (
 			MsgType: new(ssproto.Message),
 			Descriptor: &p2p.ChannelDescriptor{
 				ID:                  byte(SnapshotChannel),
-				Priority:            3,
+				Priority:            2,
```

Comment:
@erikgrinaker do I understand correctly that sending snapshots are 1/3rd less important than sending chunks? or it should be vice versa? maybe we should just assign `5` to both channels?

Reply:
Higher priority is better, right? So snapshot metadata is 3 times more important than chunks. This is because chunks are large and restoration is time-consuming, while snapshot metadata is tiny and requested in real-time, so it should have priority.

We probably want chunks to have low priority in general, for the same reason. It shouldn't have priority above e.g. consensus.

Reply:
> Higher priority is better, right?

right